file/1.5/modules/ROOT/pages/index.adoc

```diff
@@ -1,62 +1,34 @@
 = File Connector - Mule 4
 :page-aliases: connectors::file/file-connector.adoc
 
+Anypoint Connector for File (File Connector) manages files and folders on a locally mounted file system. The connector's main features include:
 
+* The ability to read files or fully list directory contents on demand.
+* Support for common File operations such as creating directories, reading, writing, copying, moving, renaming, deleting and listing files.
+* Support for locking files.
+* File matching functionality.
+* A design that is consistent with FTP Connector and SFTP Connector.
 
-File Connector v1.5
+== Representation of File Attributes
 
-Anypoint Connector for File (File Connector) handles files and folders on a locally mounted file system. Its main features include:
+Mule 4 provides a message structure of a payload and attributes in which the data moves around, making that payload a file content. The attributes provide an object that contains metadata on the payload and information such as the file’s name, path, size, and timestamps. For FTP Connector and SFTP Connector, these attributes are known as  _file attributes_.
```

Comment:
```suggestion
Mule 4 provides a message structure of a payload and attributes in which the data moves around, making that payload file content. The attributes provide an object that contains metadata on the payload and information such as the file’s name, path, size, and timestamps. For FTP Connector and SFTP Connector, these attributes are named  _file attributes_.
```

Reply:
Question on the above text. Do the attributes provide an object or are they represented as an object? I don't think "provide" is the right word.